test/test_gnmi_appldb.py

```diff
@@ -8,16 +8,16 @@
 test_data_update_normal = [
     [
         {
-            'update_path': '/sonic-db:APPL_DB/DASH_QOS',
-            'get_path': '/sonic-db:APPL_DB/_DASH_QOS',
+            'update_path': '/sonic-db:APPL_DB/localhost/DASH_QOS',
```

Comment:
Do we also use localhost in APPL_DB? I remember the localhost is only used in CONFIG_DB.

Reply:
GNMI needs to support multi-asic for APPL_DB, so I use the same schema for APPL_DB.